extensions/git/src/model.ts

```diff
@@ -232,7 +253,10 @@ export class Model {
 				return;
 			}
 
-			const repository = new Repository(this.git.open(repositoryRoot), this.globalState);
+			const repository = new Repository(
+				this.git.open(repositoryRoot, config.useRawRootPath ? path : undefined),
```

Comment:
I really don't think using `path` here makes any sense, since it can be the path of any file in the filesystem. What are you trying to solve with this PR exactly? Do you have a reproducible bug?